code/game/objects/items/research_upgrades.dm

```diff
@@ -0,0 +1,72 @@
+//prop items
+/obj/item/XenoBio
```

Comment:
WeDon'tDoCamelCase

Reply:
old item(~6 years) which is literally mapped all over in almost all maps and removing/modyfing those will bring more pain than its worth. I can probably move them to some kind of prop item folder.

Reply:
done